.yarnrc

```diff
@@ -0,0 +1 @@
+save-prefix ""
```

Comment:
😱 this is a game changer!

Reply:
You can even set it globally if you do it in `~/.yarnrc`!